apps/docs/pages/guides/getting-started/tutorials/with-expo.mdx

```diff
@@ -46,15 +47,27 @@ These variables will be exposed on the browser, and that's completely fine since
 [Row Level Security](/docs/guides/auth#row-level-security) enabled on our Database.
 
 ```ts title=lib/supabase.ts
-import AsyncStorage from '@react-native-async-storage/async-storage'
+import * as SecureStore from "expo-secure-store";
 import { createClient } from '@supabase/supabase-js'
 
+const ExpoSecureStorageAdapter = {
+  getItem: (key: string) => {
+    return SecureStore.getItemAsync(key);
+  },
+  setItem: (key: string, value: string) => {
+    SecureStore.setItemAsync(key, value);
+  },
+  removeItem: (key: string) => {
+    SecureStore.deleteItemAsync(key);
+  },
+};
+
 const supabaseUrl = YOUR_REACT_NATIVE_SUPABASE_URL
 const supabaseAnonKey = YOUR_REACT_NATIVE_SUPABASE_ANON_KEY
 
 export const supabase = createClient(supabaseUrl, supabaseAnonKey, {
   auth: {
-    storage: AsyncStorage as any,
+    storage: ExpoSecureStoreAdapter as any,
```

Comment:
Just a little typo - the variable defined on top is actually called `ExpoSecureStorageAdapter`

Reply:
Fixed this now, changed the variable name to ExpoSecureStoreAdapter as this matches the wrapped library name.